diff -uNr lrzsz-0.12.20.orig/acinclude.m4 lrzsz-0.12.20/acinclude.m4
--- lrzsz-0.12.20.orig/acinclude.m4	Sun Dec 27 23:08:59 1998
+++ lrzsz-0.12.20/acinclude.m4	Sat Sep  6 21:14:40 2003
@@ -1,7 +1,6 @@
 dnl AC_REPLACE_GNU_GETOPT
 AC_DEFUN(AC_REPLACE_GNU_GETOPT,
-[AC_CHECK_FUNC(getopt_long, , [LIBOBJS="$LIBOBJS getopt1.o getopt.o"])
-AC_SUBST(LIBOBJS)dnl
+[AC_CHECK_FUNC(getopt_long, [], [AC_LIBOBJ([getopt1]); AC_LIBOBJ([getopt])])
 ])
 
 dnl
diff -uNr lrzsz-0.12.20.orig/aclocal.m4 lrzsz-0.12.20/aclocal.m4
--- lrzsz-0.12.20.orig/aclocal.m4	Sat Sep  6 21:13:16 2003
+++ lrzsz-0.12.20/aclocal.m4	Sat Sep  6 21:14:50 2003
@@ -13,8 +13,7 @@
 
 dnl AC_REPLACE_GNU_GETOPT
 AC_DEFUN(AC_REPLACE_GNU_GETOPT,
-[AC_CHECK_FUNC(getopt_long, , [LIBOBJS="$LIBOBJS getopt1.o getopt.o"])
-AC_SUBST(LIBOBJS)dnl
+[AC_CHECK_FUNC(getopt_long, [], [AC_LIBOBJ([getopt1]); AC_LIBOBJ([getopt])])
 ])
 
 dnl
